lib/zinnia.js

```diff
@@ -276,6 +279,8 @@ const catchChildProcessExit = async ({
           maybeReportErrorToSentry(moduleErr)
         })
       }
+      // If it should be reported, it was already handled
+      err.reportToSentry = false
```

Comment:
I am a bit confused about why we are reporting the error to Sentry twice. Shouldn't we fix that instead of adding a new flag to the error object?

The condition at the top of this file checks `err.shouldReportToSentry`, but this code is setting `err.reportToSentry`.

How can we (manually) verify that the changes in the pull request work as intended?